### DISCUSSION TITLE
Добавить возможность установки кастомных конфигов для TinyMCE

### DISCUSSION THREAD

**Discussion by kozlovaa1** [moonshine-software/moonshine#587] [Ideas]
Предлагаю в управлении ресурсами добавить для поля TinyMCE возможность установить любой конфиг
```
TinyMce::make('Описание', 'body')
    ->addConfig('name', 'value'),
```
Например, чтобы[ разрешить добавление в поле любого javascript-кода](https://www.tiny.cloud/docs/tinymce/6/content-filtering/#using-extended_valid_elements-to-allow-script-elements) (по умолчанию запрещено, так как не безопасно, но мы же в админке, нам можно):
```
TinyMce::make('Описание', 'body')
    ->addConfig('extended_valid_elements', 'script[src|async|defer|type|charset]'),
```

Соответственно методом addConfig наполнять свойство config и выводить весь конфиг в data-аттрибуты, откуда конфиг заполнится в подключении скрипта в tinymce.init()

**Comment by lee-to** [comment]
Хорошая идея, сделаем

**Comment by Opekunov** [comment]
> Предлагаю в управлении ресурсами добавить для поля TinyMCE возможность установить любой конфиг

https://github.com/moonshine-software/moonshine/releases/tag/2.0.0-rc.2
